src/evaluation/mod.rs

```diff
@@ -0,0 +1,79 @@
+use std::collections::HashMap;
+
+#[derive(Clone)]
+pub struct EvaluationContext {
+    pub targetting_key: String,
```

Comment:
```suggestion
    pub targeting_key: String,
```
^ typo which will need to be updated elsewhere, not just this suggestion.

Reply:
Done